integrations/jina/src/haystack_integrations/components/embedders/jina/document_embedder.py

```diff
@@ -142,7 +158,10 @@ def _embed_batch(self, texts_to_embed: List[str], batch_size: int) -> Tuple[List
             range(0, len(texts_to_embed), batch_size), disable=not self.progress_bar, desc="Calculating embeddings"
         ):
             batch = texts_to_embed[i : i + batch_size]
-            response = self._session.post(JINA_API_URL, json={"input": batch, "model": self.model_name}).json()
+            response = self._session.post(
+                JINA_API_URL,
+                json={"input": batch, "model": self.model_name, **(parameters if parameters is not None else {})},
```

Comment:
This would be more concise, but either way works. Your call.

```suggestion
                json={"input": batch, "model": self.model_name, **(parameters or {})},
```